profiler/test/Datadog.Profiler.IntegrationTests/Helpers/EnvironmentVariables.cs

```diff
@@ -17,5 +17,6 @@ internal class EnvironmentVariables
         public const string ExceptionSampleLimit = "DD_PROFILING_EXCEPTION_SAMPLE_LIMIT";
         public const string AllocationProfilerEnabled = "DD_PROFILING_ALLOCATION_ENABLED";
         public const string ContentionProfilerEnabled = "DD_PROFILING_CONTENTION_ENABLED";
+        public const string EndpointProfilerEnabled = "DD_PROFILING_ENDPOINT_COLLECTION_ENABLED";
```

Comment:
Should we enable this in all the smoke test dockerfiles too?

Reply:
It is enabled by default. The setting is there so we can disable it if something goes wrong.